x-pack/solutions/security/packages/kbn-securitysolution-utils/src/date_math/normalize_date_math.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TimeDuration } from '../time_duration/time_duration';
+import { calcDateMathDiff } from './calc_date_math_diff';
+
+/**
+ * Normalizes date math
```

Comment:
I know we have the tests to demonstrate functionality, but perhaps we could expound here as well:

```suggestion
 * Normalizes date math strings by reducing time units, when possible
 * Example: "now-60s" -> "now-1h"
 * Example: "now-72s" -> "now-72s"
```